os/run_gdb.sh

```diff
@@ -0,0 +1 @@
+arm-none-eabi-gdb ../build/output/bin/tinyara -x gdb_commands
```

Comment:
I think this file and the below (`run_openocd.sh`) are not needed. Please remove them.